docs/fiddles/system/system-app-user-information/app-information/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  </head>
+  <body>
+    <div>
+      <div>
+        <h1>Supports: Win, macOS, Linux <span>|</span> Process: Both</h1>
+        <div>
+          <div class="demo-controls">
+            <button class="demo-button" id="app-info">View Demo</button>
+            <span class="demo-response" id="got-app-info"></span>
+          </div>
+          <p>The main process <code>app</code> module can be used to get the path at which your app is located on the user's computer.</p>
+          <p>In this example, to get that information from the renderer process, we use the <code>ipc</code> module to send a message to the main process requesting the app's path.</p>
+          <p>See the <a href="http://electron.atom.io/docs/api/app">app module documentation<span class="u-visible-to-screen-reader">(opens in new window)</span></a> for more.</p>
```

Comment:
Could you hook this `<a>` tag to use `shell.openExternal` to open it in a new window?

See what `electron-api-demos` is doing:

https://github.com/electron/electron-api-demos/blob/3890f03ecd467dd8fc47261f70ea8e409f57eff2/assets/ex-links.js#L5-L13